src/server/db/schema/products.ts

```diff
@@ -31,3 +33,22 @@ export const products = createTable(
 		categoryIdIdx: index('category_id_idx').on(table.categoryId)
 	})
 );
+
+export const productsRelations = relations(products, ({ one }) => ({
+	category: one(categories, {
```

Comment:
O que acontece com a category quando o product é removido? Precisamos colocar algo aqui?

Reply:
Voce diz no sentido de que, se o produto for deletado, ele precisa perder esse relacionamento?